docs/tutorial.md

```diff
@@ -0,0 +1,694 @@
+
+You can also see that the inferred tree sequence has fewer trees than the original,
+simulated one. There are two reasons for this. First, there are some tree changes that
+do not involve a change of topology (e.g. the first two trees in the original tree
+sequence are topologically identical, as are two trees after that).
+Even if we had different mutations on the
+different edges in these trees, such tree changes are unlikely to be spotted.
+Second, our inference depends on the mutational information that is present. If no
+mutations fall on a particular edge in the tree sequence, then we have no way of
+inferring that this edge existed. The fact that there are no mutations in the first,
+third, and fifth trees shows that tree changes can occur without associated mutations.
+As a result, there will be tree transitions that we cannot pick up. In the simulation
+that we performed, the mutation rate is equal to the recombination rate, and so we
+expect that many recombinations will be invisible to us.
+
+For similar reasons, there will be many nodes in the tree at which
+polytomies occur. Here we correctly infer that sample nodes 0 and 4 group together,
+as do 1 and 2. However, in the middle inferred tree we were not able to distinguish
+whether node 3 was closer to node 1 or 2, so we have three children of node 7 in that
+tree.
+
+Finally, you should note that inference does not always get the right answer. At the
+first site in the inferred trees, node 5 is places as an outgroup, but at this site
+it is actually closer to the group consisting of nodes 0 and 4.
+
+:::{note}
+Other than the sample node IDs, it is meaningless to compare node numbers in the
+source and inferred tree sequences.
+:::
+
+(sec_tutorial_data_example)=
+
+## Data example
+
+Inputting real data for inference is similar in principle to the examples above:
+you simply iterate over the sites in your data file, calling {meth}`.SampleData.add_site`
+for each variable site. We do not provide methods to read directly from other formats, as
+these formats are often too complex for automatic conversion. However, many Python
+libraries are available for reading different genomic data formats, and writing a script
+to iterate over the sites in such files is usually a fairly simple task. The example
+below shows how this can be done for a VCF file, using a freely available VCF reading
+library.
+
+(sec_tutorial_read_vcf)=
+
+### Reading a VCF
+
+A common way to store genetic variation data is as a VCF (Variant Call Format) file.
+The [CYVCF2 library](https://github.com/brentp/cyvcf2) provides a relatively fast
+Python interface for reading VCF files. The following script demonstrates how to infer a
+tree sequence from a simple VCF file. In particular the function below named
+`add_diploid_sites` illustrates how to iterate over the variants in a `CYVCF2.VCF`
+object and add them to a `tsinfer`` sample data file. This particular script assumes
```

Comment:
```suggestion
object and add them to a ``tsinfer`` sample data file. This particular script assumes
```

Reply:
New version pushed. Thanks for spotting the typo @jeromekelleher